Client/Frontend/Browser/BackForwardTableViewCell.swift

```diff
@@ -50,6 +50,11 @@ class BackForwardTableViewCell: UITableViewCell {
         didSet {
             if let s = site {
                 faviconView.setFavicon(forSite: s, onCompletion: { [weak self] (color, url) in
+                    if s.tileURL.isLocal {
+                        self?.faviconView.image = UIImage(named: "faviconFox" )
+                        self?.faviconView.backgroundColor = UIColor.white
```

Comment:
I added the return here, however this means avoiding scaling the icon.. Is that correct? Or were you thinking of placing the return statement somewhere else?